pkg/api/authz.go

```diff
@@ -184,10 +189,19 @@ func AuthzHandler(ctlr *Controller) mux.MiddlewareFunc {
 			acCtrlr := NewAccessController(ctlr.Config)
 
 			// allow anonymous authz if no authn present and only default policies are present
-			username := ""
+			var username string
+			var err error
 
-			if isAuthnEnabled(ctlr.Config) {
-				username = getUsername(request)
+			/* To do: verify client certs and get its username(subject DN)
```

Comment:
The keyword to use is either TODO or FIXME

Reply:
examples/metrics/kubernetes/prometheus/bundle.yaml:3762:                                      TODO: Add other useful fields. apiVersion, kind

like so ^

Reply:
"TODO" would make lint fail when updating the PR. I replaced it with "To be implemented" if it's alright with you